cmd/rootMain.go

```diff
@@ -23,7 +23,7 @@ func writeAlias(path, subcommand string) error {
 	if err != nil {
 		return err
 	}
-	_, err = f.WriteString("#!/bin/sh\nexec wash " + subcommand + " \"$@\"")
+	_, err = f.WriteString("#!/bin/sh\nWASH_EMBEDDED=1 exec wash " + subcommand + " \"$@\"")
```

Comment:
What does `WASH_EMBEDDED` mean? Is it equivalent to `IN_WASH_SHELL` ?

Reply:
Yes.

Reply:
Ok. A small comment explaining what it's for could help (so someone doesn't have to read through the code and figure out themselves that we're only doing this for more user-friendly help when running the `wash shell` vs without). Could maybe go in the usage template?

Reply:
I'm not sure it needs to be presented to people using Wash. I don't think they draw a connection between `wash help` and `help` within the `wash` shell.

Reply:
Oh, the comment was for maintainers, not users.